### PR TITLE
Update expectations_xl.R

### PR DESCRIPTION
**Pull request by markvanderloo**
Added documentation for `strict`, which is a new parameter for `expect_warning` and `expect_message`

### CODE REVIEW

File: R/expectations_xl.R [hunk 1]
```diff
@@ -20,6 +20,8 @@
 #' @param format \code{[character]} Comparison mode passed to \code{diffPrint},
 #' defaults to to using the \dQuote{diffobj.format} global option value with
 #' \dQuote{ansi256} as fallback if no such option is set
+#' @param strict \code{[logical]} scalar. If set to \code{TRUE}, any exception 
+#'        worse than the wanted exception will cause the test to fail.
```
Comment (eddelbuettel) [inline R/expectations_xl.R]: Methinks you forgot to a) add it the signature of the function(s) and b) possibly the calls and c) the generated manual page (I can do the lattter if roxygen shows side effects of rewriting).

Comment (eddelbuettel) [reply R/expectations_xl.R]: It also lacks a default value.  Should it be `TRUE` in the call?  Or another value?  Please advise.

**Comment by eddelbuettel** [comment]
@markvanderloo I cannot assign you a code review on PR you started but please have a look at what I just added.

File: R/expectations_xl.R (hunk 1 above)
Comment (markvanderloo) [reply R/expectations_xl.R]: The default value is `FALSE` in `tinytest` (for back-compatibility). I was under the (possibly false) impression that the signature and default values are automatically imported via the call to `supersize`

Comment (markvanderloo) [reply R/expectations_xl.R]: I think I should have been more explicit. The `strict` argument was only added to `expect_warning` and `expect_message`. So no need to add them to `expect_equivalent` etc. As these are imported through `supersize` I _think_ this is all automatic. I'll do a local change and test against the development version of `tinytest`

Comment (eddelbuettel) [reply R/expectations_xl.R]: Yes, I corrected the PR for this hours ago [in this commit](https://github.com/eddelbuettel/ttdo/pull/8/commits/5c193ce725c24bffe45f6e52ce6683a5a36008c7See) 

I also realized later that we probably could have added this simply (at your end) as a simple argument `strict` without a value follow by `if (missing(strict)) strict <- FAlSE` which might have been smoother upgrade path.  Too late now (and I didn't work through any details).  Anyway.

**Comment by eddelbuettel** [comment]
Once we properly review the issue, as is the best of my knowledge fairly common when a downstream dependency changes and requests changes of its reverse-dependencies, we see that the upcoming `tinytest` adds a new argument:

```
[...]
✔  checking for missing documentation entries ...
W  checking for code/documentation mismatches ...
   Codoc mismatches from documentation object 'ttdo_boolean_and_message_tests':
   expect_warning_xl
     Code: function(current, pattern = ".*", class = "warning", info =
                    NA_character_, strict = FALSE, ...)
     Docs: function(current, pattern = ".*", class = "warning", info =
                    NA_character_, ...)
     Argument names in code not in docs:
       strict
     Mismatches in argument names:
       Position: 5 Code: strict Docs: ...
   expect_message_xl
     Code: function(current, pattern = ".*", class = "message", info =
                    NA_character_, strict = FALSE, ...)
     Docs: function(current, pattern = ".*", class = "message", info =
                    NA_character_, ...)
     Argument names in code not in docs:
       strict
     Mismatches in argument names:
       Position: 5 Code: strict Docs: ...
   
✔  checking Rd \usage sections (1s)
[...]
```

So commit https://github.com/eddelbuettel/ttdo/pull/8/commits/b42baefd444f58bc36a05ef19495e213a6e35f94 was in the wrong place.   Upcoming commit will fix this.

**Pull request closed without merging**